geomesa-filter/src/main/scala/org/locationtech/geomesa/filter/FilterHelper.scala

```diff
@@ -106,7 +106,7 @@ object FilterHelper extends LazyLogging {
     }
   }
 
-  private def recreateAsIdlSafeFilter(op: BinarySpatialOperator,
+  def recreateAsIdlSafeFilter(op: BinarySpatialOperator,
```

Comment:
I think this change might have been from intermediate work.  Let's try and revert the changes in this file.